papermill/engines.py

```diff
@@ -34,6 +35,11 @@ def __init__(self):
     def register(self, name, engine):
         self._engines[name] = engine
 
+    def register_entry_points(self):
```

Comment:
This needs calling at the end of this file I think so that entrypoints actually get added.